buildenv/jenkins/JenkinsfileBase

```diff
@@ -59,7 +59,7 @@ def setupEnv() {
 	CLOUD_PROVIDER = params.CLOUD_PROVIDER ? params.CLOUD_PROVIDER : ""
 	env.JDK_BRANCH = params.JDK_BRANCH ? params.JDK_BRANCH : ""
 	env.USER_CREDENTIALS_ID = params.USER_CREDENTIALS_ID ? params.USER_CREDENTIALS_ID : ""
-	env.TEST_JDK_HOME = "$WORKSPACE/openjdkbinary/j2sdk-image"
+	env.TEST_JDK_HOME = properties.TEST_JDK_HOME
```

Comment:
Setting `TEST_JDK_HOME` as `properties.TEST_JDK_HOME` is very confusing. We can just remove line62.